### DISCUSSION TITLE
能翻译但不能朗读（TTS连接超时）

### DISCUSSION THREAD

**Discussion by RoninLJY** [YiiGuxing/TranslationPlugin#3773] [Q&A]
使用有道翻译，能成功翻译，但是无法进行朗读。显示TTS连接超时.请问大家有遇到过吗？
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/82429213/1f921483-f4f3-4448-b0cd-b40a1e25f61d)

**Comment by YiiGuxing** [comment]
见：#2315

**Reply by RoninLJY** [reply]
非常感谢！！ Star爆满！！！
